nevergrad/optimization/optimizerlib.py

```diff
@@ -95,13 +95,28 @@ def _internal_ask_candidate(self) -> p.Parameter:
                                      mutator.get_roulette(self.archive, num=2))
             return pessimistic.set_standardized_data(data, reference=ref)
         # mutating
+<<<<<<< HEAD
```

Comment:
here you have a conflict between versions, which should be easy to solve -- just come back to the current master.